tox.ini

```diff
@@ -46,3 +46,39 @@ commands =
     postgres: py.test --ds=pytest_django_test.settings_postgres --strict -r fEsxXw {posargs:tests}
     sqlite: py.test --ds=pytest_django_test.settings_sqlite --strict -r fEsxXw {posargs:tests}
     sqlite_file: py.test --ds=pytest_django_test.settings_sqlite_file --strict -r fEsxXw {posargs:tests}
+
+[testenv:doc8]
+basepython = python3.6
+skip_install = true
+deps =
+    sphinx
+    doc8
+commands =
+    doc8 docs/
+
+[testenv:readme]
+basepython = python3.5
```

Comment:
Why only 3.5 here (and below)?
Could also be 3.6?!

Reply:
Twine 1.9.1 doesn't officially support 3.6, as well as some of the other tools. I set all the max versions with which I have worked with.